tooling/api/src/window.ts

```diff
@@ -1096,6 +1096,10 @@ class WindowManager extends WebviewWindowHandle {
 
 /**
  * Create new webview windows and get a handle to existing ones.
+ *
+ * Windows are identified by a *label*  a unique identifier that can be used to reference it later.
+ * It may only contain alphanumeric characters: `a-zA-Z-/:_`.
```

Comment:
i'm not 100% happy with that phrasing since `-/:_` is not part of the alphanumeric subset, it's more like alphanumeric _plus_ these symbols.
But it's not that bad, so i could live with it 🤷 

Same for the other comment btw

Reply:
Check latest version, I adapted your comment on the original issue to make the main doc comment more helpful. I want to defer the meat of documentation to the main comment but I could update the second comment too

Reply:
i think it's fine to keep the other comment short. Maybe a little hint that it needs to be unique tho? Something like `The unique webview window label`? If that's too long we can probably cut the "webview" here 🤔

Reply:
good point, IMO we drop the webview and add `unique`